templates/user/confirm.html.twig

```diff
@@ -0,0 +1,9 @@
+{% extends '@SumoCodersFrameworkCore/user.html.twig' %}
+
+{% block main %}
+    <p>{{ 'Confirm your account to continue.'|trans }}</p>
```

Comment:
Kan je eens screenshot van deze pagina maken aub. Want kan me niet voorstellen hoe die er nu uit ziet.

Reply:
@tijsverkoyen Dit is een voorbeeld voor bv dpg media
![Screenshot 2024-03-26 at 11 14 40](https://github.com/sumocoders/Framework-User-Implementation-Example/assets/3007416/018491bb-919e-436f-81f7-b181dac5697e)

Reply:
Die Confirm mag nog vertaald worden naar "Bevestig je account"

Reply:
fixed #c1221fd586b42ee001c54ae311bf85b169a120b5